packages/legend-query-builder/src/stores/QueryBuilderState.ts

```diff
@@ -253,6 +253,7 @@ export abstract class QueryBuilderState {
   changeMapping(val: Mapping): void {
     this.resetQueryResult();
     this.resetQueryContent();
+    this.milestoningState.updateMilestoningConfiguration();
```

Comment:
I will move this statement one line down, but I totally missed the fact that we need to call this again post calling `resetQueryContent`. Thanks!